### DISCUSSION TITLE
set heuristicCoefficient via UI

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#13602] [Q&A]
Am I right that THIS ist NOT working in a profile? Use such a profile seams an endless-loop ;(

`<parameter id="Heuristic" name="Heuristic" description="Heuristik-Koeffizient" 
type="numeric" 	
values="1,1.2,1.3,1.4,1.5,1.6"
valueDescriptions="1,1.2,1.3,1.4,1.5,1.6"
/>`

`<attribute name="heuristicCoefficient" value=":Heuristic" />`

**Comment by vshcherb** [comment]
I'm afraid it doesn't work, you probably can only create different profiles and extending default profiles.
